docs/configuration/configuration.md

```diff
@@ -874,8 +874,12 @@ tls_config:
 
 By default every app listed in Marathon will be scraped by Prometheus. If not all
 of your services provide Prometheus metrics, you can use a Marathon label and
-Prometheus relabeling to control which instances will actually be scraped. Also
-by default all apps will show up as a single job in Prometheus (the one specified
+Prometheus relabeling to control which instances will actually be scraped.
+See [the Prometheus marathon-sd configuration file](/documentation/examples/prometheus-marathon.yml)
```

Comment:
@simonpasquier @grobie do you know if this link would actually work in the docs?

Reply:
It should work as it is already used this way in the `kubernetes_sd_config` section:
https://github.com/prometheus/prometheus/blame/36143be2349f414623a45ffe6143bc9ae71cb4d4/docs/configuration/configuration.md#L804

Reply:
Ack. Will merge and cherrypick into release branch, as I feel it should in there.